### ISSUE TITLE
GRPO Hangs

### ISSUE THREAD

**Issue by WeizhenWang-1210** [hiyouga/EasyR1#231]
I set up my environments using Anaconda on an Ubuntu machine. However, when I use
```
bash examples/qwen2_5_vl_3b_geo3k_grpo.sh
```
The process seems to be frozen, with 100% GPU utilization but only 1.5GB vRAM occupation.
I wonder if people have been successful in running this script inside Docker environments.

**Comment by Tree-Shu-Zhao** [comment]
Hi. I met the same issue. After debugging, I found my code stuck at actor loss.backward(). Could you please share your solution?
